dev-client/src/components/sheets/StandaloneOverlaySheet.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {forwardRef} from 'react';
+
+import {
+  BottomSheetScrollView,
+  BottomSheetModal as GorhomBottomSheetModal,
+} from '@gorhom/bottom-sheet';
+import {useFocusEffect} from '@react-navigation/native';
+
+import {BackdropComponent} from 'terraso-mobile-client/components/BackdropComponent';
+import {
+  ModalContext,
+  ModalHandle,
+  ModalTrigger,
+} from 'terraso-mobile-client/components/modals/Modal';
+import {useGorhomModalHandleRef} from 'terraso-mobile-client/hooks/gorhomHooks';
+import {useHeaderHeight} from 'terraso-mobile-client/hooks/useHeaderHeight';
+
+export type StandaloneOverlaySheetProps = React.PropsWithChildren<{
+  trigger?: ModalTrigger;
+}>;
+
+/*
+ * Simple overlay sheet with no special header or sizing. Just fits to its
+ * contents, making them scrollable if needed. Intended for one-off cases
+ * where design doesn't indicate more specific sheet behavior.
+ */
+export const StandaloneOverlaySheet = forwardRef<
```

Comment:
This is functionally a one-off case that handles `ImagePicker`; it fits neither of the other two use-cases, and it seemed cleaner to encapsulate the guts of interacting with `GorhomBottomSheetModal` without implying a broader design. My discussion with @CourtneyLee333 indicated that some application components may just be these simple standalone sheets, so this component exists to serve such cases.

Reply:
I don't think so. There's nothing about it that's specific to that component and there may be other similar cases in the future.